js/_website/src/routes/custom-components/gallery/+page.svelte

```diff
@@ -7,7 +7,7 @@
 
 	const API = "https://gradio-custom-component-gallery-backend.hf.space/";
 	const OFFSET = 0;
-	const LIMIT = 50;
```

Comment:
Why have any limit at all?

Reply:
Thought it'd be better to paginate the results but I guess we can remove since we're not at that scale